scripts/generate_bash_completion_for_runner

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+
+readonly BASH_COMPLETION_DIR="$(realpath ".")"
+readonly BASH_COMPLETION_FILE=$BASH_COMPLETION_DIR/.runner_bash_completion 
+
+./scripts/runner completion --out-dir="$BASH_COMPLETION_DIR"
```

Comment:
Is it ok to do all this work in bashrc? IIUC it is compiling and running runner every time a new bash shell is created. I thought you suggested another approach in the past, of checking in the completion file and just sourcing that, is that not an option any more?